cmake/onnxruntime_mlas.cmake

```diff
@@ -568,3 +588,40 @@ if (NOT onnxruntime_BUILD_SHARED_LIB)
             RUNTIME   DESTINATION ${CMAKE_INSTALL_BINDIR}
             FRAMEWORK DESTINATION ${CMAKE_INSTALL_BINDIR})
 endif()
+
+
+if (NOT onnxruntime_ORT_MINIMAL_BUILD)
+  onnxruntime_add_executable(onnxruntime_mlas_q4dq
```

Comment:
might be useful to have a comment describing what this is. it is not obvious to me from the name.